openfisca_core/parameters.py

```diff
@@ -501,7 +503,7 @@ def add_child(self, name, child):
         :param child: The new child, an instance of :any:`Scale` or :any:`Parameter` or :any:`ParameterNode`.
         """
         if name in self.children:
-            raise ValueError("{} has already a child named {}".format(self.name, name)).encode('utf-8')
+            raise ValueError("{} has already a child named {}".format(self.name, name).encode('utf-8'))
```

Comment:
Because `ValueError(string).encode('utf-8')` doesn't make sense (you can encode a string, not an exception).


`ValueError(string.encode('utf-8'))` makes sense